pages/consultation/index.tsx

```diff
@@ -66,24 +81,27 @@ const ConsultationPage: NextPage = () => {
             <FormSection title={CONTACT_INFORMATION}>
                 <ContactInformation 
                     setContactInformation={setContactInformation}
+                    setContactInformationError={setContactInformationError}
                 />
             </FormSection>
             <FormSection title={BUDGET_AND_TIMELINE}>
                 <BudgetTimelineForm
                     setBudget={setBudget}
                     setTimeline={setTimeline}
                     defaultBudget={[DEFAULT_MIN_BUDGET, DEFAULT_MAX_BUDGET]}
+                    setBudgedTimeLineError={setBudgedTimeLineError}
                 />
             </FormSection>
             <FormSection title={PROJECT_GOALS}>
                 <ProjectGoalsForm 
                     setServiceInformation={setServiceInformation}
                     setServices={setServices}
+                    setServiceInformationError={setServiceInformationError}
                 />
             </FormSection>
             <Section>
                 <div onClick={(e: any) => handleSubmit(e)}>
-                    <Button variant='tertiary'>
+                    <Button variant='tertiary' disabled={disabledButton}>
                         {SUBMIT}
                     </Button>
                 </div>
```

Comment:
Submit is still getting called even when the form is incorrect 🥴

Reply:
Yup, that's because of the button comoponent, see that he onClick is in the div and not in the button

Reply:
Ik, but it shouldn't get sent, pls fix